exporter/collector/config.go

```diff
@@ -59,6 +59,20 @@ type MetricConfig struct {
 	// Buffer size for the channel which asynchronously calls CreateMetricDescriptor. Default
 	// is 10.
 	CreateMetricDescriptorBufferSize int `mapstructure:"create_metric_descriptor_buffer_size"`
+
+	// If true, the exporter will copy OTel's service.name, service.namespace, and
+	// service.instance.id resource attributes into the GCM timeseries metric labels. This
+	// option is recommended to avoid writing duplicate timeseries against the same monitored
+	// resource. Default is true.
+	IncludeServiceResourceAttributes bool `mapstructure:"include_service_resource_attributes"`
+
+	// If provided, resources attributes matching any filter will be included in metric labels.
```

Comment:
Can you expand the comment to say what the default value is and how it interacts with include service resource attributes? (e.g. does setting resource filter to an empty list remove the service attributes above?)